### ISSUE TITLE
qRANSAC_SD license is not compatible with GPL

### ISSUE THREAD

**Issue by tpwrules** [CloudCompare/CloudCompare#1787]
The qRANSAC_SD license places restrictions on usage which are not compatible with the GPL:

https://github.com/CloudCompare/CloudCompare/blob/b843609b84f4a2617ef861b5bab5d8cb41e0e97b/plugins/core/Standard/qRANSAC_SD/extern/RANSAC_SD/ReadMe.txt#L33-L54

This should be made clear and the plugin probably disabled for official distributions. I haven't checked if other plugins have similar restrictions.

**Comment by rjanvier** [comment]
Yes, it is a known issue. For the Linux side, it's why it was disabled for Debian package and snap package (I don't know for flatpack). As far as I know, in April-may 2020 it was the only non [DFSG](https://wiki.debian.org/DebianFreeSoftwareGuidelines) compatible source code in CC.

**Comment by robot-army** [comment]
The documentation here should probably be updated too. I found details on how to use this, but it took me a while to see that it's disabled by default. https://www.cloudcompare.org/doc/wiki/index.php/RANSAC_Shape_Detection_(plugin)
